Mathlib/Analysis/Normed/Group/SeparationQuotient.lean

```diff
@@ -20,11 +20,11 @@ All the following definitions are in the `SeparationQuotient` namespace. Hence w
 
 * `normedMk` : the normed group hom from `M` to `SeparationQuotient M`.
 
-* `liftNormedAddGroupHom` :
-Any bounded group hom `f : M → N` such that `∀ x, ‖x‖ = 0 → f x = 0` descends to a bounded group hom
-`SeparationQuotient M → N`. Here, `(f : NormedAddGroupHom M N)`, `(hf : ∀ x : M, ‖x‖ = 0 → f x = 0)`
-and `liftNormedAddGroupHom f hf : NormedAddGroupHom (SeparationQuotient M) N` such that
-`liftNormedAddGroupHom f hf (mk x) = f x`.
+* `liftNormedAddGroupHom` : any bounded group hom `f : M → N` such that `∀ x, ‖x‖ = 0 → f x = 0`
```

Comment:
Unrelated comment, there's a mix between ` : ` and `: ` throughout mathlib, with the latter being quite a bit more common, but not uniformly so. Do you think it's worth standardising?

Reply:
I like standardising such things, but I'm not sure how much that's me versus being universally useful. (I haven't seen bikesheds about this question, for instance.) But feel free to start a discussion on zulip!

Reply:
I think you and I are on the same page here, but I agree it's not clear if this one is actually useful! It's probably easiest just to leave this one for now?